### DISCUSSION TITLE
The Future of Bromite?

### DISCUSSION THREAD

**Discussion by peternrdstrm** [bromite/bromite#2044] [General]
Over the last year, Bromite has improoved a lot. Just take a look at it. Userscript support, bottom navbar, no history mode, and so on and so on. As Bromite rises though, I wonder what is held for it's future. 
What are your long-term plans with it?
Also, on the user side, what would you guys like to see?

**Comment by lipici** [comment]
1) To be in the official repo of f-droid, im waiting a little bit more, if not i will make a decission if i use it or not

2) Keeping up to date with privacy, 100%  open source, no ads,  2 versions in which one its paid version to suport developer, program to promote the browser ( everyone to send mail to x tech site to review bromite, to make tutorial how to install, benefits ) , extensions, direct menu for long screenshoot, 3 last app for sharing instead of one.

**Comment by csagan5** [comment]
> Userscript support, bottom navbar, no history mode, and so on

All of this has been the work of @uazo which I have reviewed and accepted (in most cases) after some discussion and testing.

> I wonder what is held for it's future. What are your long-term plans with it?

No plans, it is an open source project and we work on it in our free time. This is not a product that makes money for anybody.

My wish is that it stays open source and true to its mission about privacy and user freedom so I will stick to that for the time being.

**Reply by uazo** [reply]
> All of this has been the work of @uazo

it's a pleasure for me to contribute! and thanks for the trust

> My wish is that it stays open source

mine is that there were more people involved in the project.
unfortunately @csagan5 I could not see your answer to my question, how did you do everything by yourself!

**Comment by kgwooo** [comment]
Thanks so much for your efforts! A few years back, I did contribute a small amount towards the effort...
